### DISCUSSION TITLE
Why does Polaris use rem for all properties?

### DISCUSSION THREAD

**Discussion by TKDev7** [Shopify/polaris-react#11752] [General]
I am trying to understand why would you use rem for everything and not only font related properties. Like why use rem for box-shadow for example?

**Comment by kyledurand** [comment]
If this is meant to be a discussion can you add some context or propose alternatives? There are plenty of examples of why rem is better than px / em but opinions can go [back and forth](https://stackoverflow.com/questions/11799236/should-i-use-px-or-rem-value-units-in-my-css)

**Reply by TKDev7** [reply]
I am trying to understand why you would want your box shadow to scale with root font size
